presto-docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -159,6 +159,19 @@ For example, the mapping list might have URL prefix ``s3://abc/xyz/`` followed b
 than for other paths within the bucket. You can set default configuration by not
 including any match criteria for the last entry in the list.
 
+In addition to the rules above, the default mapping can contain the
+``userClusterDefault`` property set to ``true``. This causes the the default
+cluster role to be used as a fallback option. It can not be used with the
+following configuration properties:
+
+- ``accessKey``
+- ``secretKey``
+- ``iamRole``
+- ``allowedIamRoles``
+
+If no mapping entry matches and no default is configured, an error
+``AccessDeniedException: Access Denied: No matching S3 security mapping`` occurs.
```

Comment:
I don't think we should document exact error message. It can easily make docs obsolete.

Reply:
done.